bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/Semester.scala

```diff
@@ -49,7 +49,7 @@ object Semester {
 
 }
 
-case class Semester(year: Int, half: SemesterOption) {
+sealed case class Semester(year: Int, half: SemesterOption) {
```

Comment:
Did you mean `final`?

Reply:
I guess it should be both. we have defined 2 instances A/B

Reply:
I got confused with `SemesterOption`, yes it should be `final`